source/electrical_heating/src/opengeh_electrical_heating/domain/calculation.py

```diff
@@ -13,6 +13,13 @@
 from opengeh_electrical_heating.domain.column_names import ColumnNames
 from opengeh_electrical_heating.domain.types import NetSettlementGroup
 from opengeh_electrical_heating.domain.types.metering_point_type import MeteringPointType
+from opengeh_electrical_heating.infrastructure.electricity_market.child_metering_points.wrapper import (
+    ChildMeteringPoints,
+)
+from opengeh_electrical_heating.infrastructure.electricity_market.consumption_metering_point_periods.wrapper import (
+    ConsumptionMeteringPointPeriods,
+)
+from opengeh_electrical_heating.infrastructure.measurements.measurements_gold.wrapper import TimeSeriesPoints
```

Comment:
I've added these public symbols to `infrastructure/__init__.py` to avoid having to know the inner structure when used from outside `infrastructure`. It also adds the benefit of greatly simplified imports.
```suggestion
from opengeh_electrical_heating.infrastructure import (
    ChildMeteringPoints,
    ConsumptionMeteringPointPeriods,
    TimeSeriesPoints,
)
```